sycl/doc/extensions/experimental/sycl_ext_oneapi_graph.asciidoc

```diff
@@ -556,6 +556,98 @@ Parameters:
 
 |===
 
+==== Dynamic Command Groups
+
+[source,c++]
+----
+namespace ext::oneapi::experimental{
+class dynamic_command_group {
+public:
+  dynamic_command_group(
+      command_graph<graph_state::modifiable> graph, std::vector<std::function<void(handler &)>> cgfList);
```

Comment:
Should the vector be a const ref? Or is there a reason we think the runtime would need to modify it.

I think we also talked about using a parameter pack for this.

Reply:
Yeah, `const ref` makes sense to me

I changed this due to team discussions about how intuitive using indexes is with parameter packs. When using a vector, which indexes map to which cgf is more clearly defined.

That said, I don't have a very strong preference on this and I'm happy to change it back to a parameter pack if that's the consensus.